app/policies/worksheet_review_policy.rb

```diff
@@ -0,0 +1,17 @@
+class WorksheetReviewPolicy < ApplicationPolicy
+
+  def create?
+    %w[admin teacher].include?(@user.role)
```

Comment:
this would mean that a teacher can create a comment in any worksheet, even those that he doesn't "own"